docs/jump/README.md

```diff
@@ -34,17 +34,15 @@ to synthesize new optimization paradigms!
 can be installed by entering the following in the REPL.
 
 ```julia
-julia> ]
-
-(v1.8) pkg> add InfiniteOpt
+julia> import Pkg; Pkg.add("InfiniteOpt")
 ```
 
 ## Documentation
 [![](https://img.shields.io/badge/docs-stable-blue.svg)](https://infiniteopt.github.io/InfiniteOpt.jl/stable)
```

Comment:
I wonder if you want to remove some of these badges. They don't add anything that isn't covered by the link below.

Reply:
Done